python/ray/tune/tests/execution/test_controller_resume_integration.py

```diff
@@ -201,16 +200,20 @@ def test_controller_restore_error_only_resume(
     new_runner = TuneController(
         resource_manager_factory=lambda: resource_manager_cls(),
         storage=STORAGE,
-        resume="ERRORED_ONLY",
+        resume_config=ResumeConfig(
```

Comment:
Why do we need to use `ResumeConfig` now? It seems this class is not introduced in this PR.

Reply:
this test was actually not run before [this change](https://github.com/ray-project/ray/pull/46905/files#diff-5a9919a777b978fee388f8d0477558ecb309ade3bdb68f0cb400a13be873cdde), so I had to fix a bunch of things that have changed since the test was written.